components/02-molecules/menu/menu.twig

```diff
@@ -34,15 +34,19 @@
 
 {% set menu__attributes = menu__attributes|merge({
   'id': menu__base_class,
-  'class': bem(menu__base_class, menu__blockname, menu__modifiers, menu__additional_classes),
+  'class': bem(menu__base_class, menu__modifiers, menu__blockname, menu__additional_classes),
   'aria-label': menu__name,
 }) %}
 
+{% if breadcrumb %}
```

Comment:
See comment above. I think we need to avoid checking for specific components. In this case it seems like having a variable `ordered_menu = TRUE` or something would be more generic and applicable for multiple components.

Reply:
Updated.